src/store/modules/Configuration/NetworkSettingsStore.js

```diff
@@ -0,0 +1,60 @@
+import api from '../../api';
+
+const NetworkSettingsStore = {
+  namespaced: true,
+  state: {
+    networkData: null,
+    ipAddress: '--',
+    macAddress: '--'
+  },
+  getters: {
+    networkData(state) {
+      return state.networkData;
+    },
+    ipAddress(state) {
+      return state.ipAddress;
+    },
+    macAddress(state) {
+      return state.macAddress;
+    }
+  },
+  mutations: {
+    setNetworkData(state, networkData) {
+      state.networkData = networkData;
+    },
+    setIpAddress(state, ipAddress) {
+      state.ipAddress = ipAddress;
+    },
+    setMacAddress(state, macAddress) {
+      state.macAddress = macAddress;
+    }
+  },
+  actions: {
+    getNetworkData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/network/enumerate')
+        .then(({ data }) => {
+          const networkData = data.data;
```

Comment:
Are we going to use object destructuring or maintain consistency in showing the response's data structure? I feel like its more readable when not using object destructuring.